core-feature-pack/src/main/resources/content/bin/standalone.sh

```diff
@@ -82,7 +82,7 @@ if $cygwin ; then
 fi
 
 # Setup JBOSS_HOME
-RESOLVED_JBOSS_HOME=`cd "$DIRNAME/.."; pwd`
+RESOLVED_JBOSS_HOME=`cd "$DIRNAME/.." >/dev/null; pwd`
```

Comment:
What does this change do to fix the problem?

Not including any output from 'cd' in the value of RESOLVED_JBOSS_HOME makes sense, but I'm not clear on what CDPATH has to do with this, other than that 'cd' might produce output if CDPATH means it can't change to the directory correctly. But then that's an error condition too.